_shared/project/tox.ini

```diff
@@ -84,6 +86,7 @@ deps =
         {{- include("tox/deps", indent=4) -}}
     {% endif %}
 depends =
+    tests: clean
```

Comment:
Make the `tests` environment depend on the `clean` environment.

This means that whenever a single tox command runs both `tests` and `clean` (e.g. `tox -e tests,clean` or `tox --parallel -e tests,clean`) tox will re-order the environments to run `clean` before `tests`.

This matches all `tests` environments, e.g. `tests`, `py312-tests`, `py311-tests`, etc.